package.json

```diff
@@ -53,5 +53,11 @@
   },
   "publishConfig": {
     "access": "public"
+  },
+  "c8": {
+    "exclude": [
```

Comment:
Is this necessary? I think removing this would still make the tests pass, no?

Reply:
@gurgunday I removed it. The tests pass, but it looks like the c8 code coverage is also checking the examples folder. I'm not sure if the CI can pass.